core/templates/pages/topics-and-skills-dashboard-page/topics-list/topics-list.directive.html

```diff
@@ -96,7 +113,7 @@
     width: 90%;
   }
   topics-list .topic-edit-options {
-    background-color: white;
+    background-color: #ffffff;
     border: 1px solid black;
```

Comment:
Change this as well. There seem to be few more in this file.

Reply:
Done.